velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -338,5 +338,38 @@ TEST_F(ArithmeticTest, hypot) {
   EXPECT_DOUBLE_EQ(5.70087712549569, hypot(3.5, -4.5).value());
 }
 
+class LogNTest : public SparkFunctionBaseTest {
+ protected:
+  std::optional<double> log2(std::optional<double> arg) {
```

Comment:
Can we remove the unused functions here?

Reply:
Thanks, Just updated.